Add mtucker's Boston file

markrtucker opened the pull request:
Added a few places to get great food in Boston.

markrtucker committed:
Add mtucker's Boston file

Added a few places to get great food in Boston.

githubteacher commented:
@mtucker01752 do you have any suggestions for places to find awesome local 🍻 ?

Hooray Boston!!! ✨ 
![i](http://i.giphy.com/lgK3c8EsnQnGE.gif)

markrtucker commented:
@githubteacher Tavern in the Square has lots of local micro-brews on tap. Or check out Five Horses Tavern (http://www.fivehorsestavern.com) for something a bit different.

vicmaster commented on mtucker01752-boston.md:
```diff
@@ -0,0 +1,7 @@
+## Boston, MA
+
+### Places to eat
+- Tavern in the Square - http://taverninthesquare.com
```
What about to use markdown for links like?:
- [Tavern in the Square](http://taverninthesquare.com)

What do you think?

markrtucker replied:
@vicmaster - great idea! I edit the file locally, now to figure out how to make github recognize the changes....

vicmaster replied:
Done :)

markrtucker committed:
mtucker trying again

another commit...

  Please enter the commit message for your changes. Lines starting

markrtucker committed:
Update mtucker01752-boston.md

Made a comment to test out editing in the browser

markrtucker committed:
Add locations to mtucker's Boston file

added some locations, including the opera house and the garden

markrtucker commented:
Closes #11
